boards/FSM/Inc/App/App_WheelSpeedSensor.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+struct WheelSpeedSensor;
+
+enum WheelSpeedFaultStatus
```

Comment:
I would remove this enum entirely and use `#include "ErrorCode.h"`

Reply:
removed this after https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582#discussion_r417063611

Reply:
Got rid after https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/582#discussion_r417060913